src/client/components/componentWithSliders/modalsavewindow/modalSaveWindow.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './modalSaveWindow.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+
+const cancelWindowIcon = <FontAwesomeIcon icon={faTimes}/>;
+
+const SavePressetesModalWindow = (props) => (
+        <div className="modalSaveWindow">
+            <div className="headerModalWindow">
+                {cancelWindowIcon} 
+            </div>
+            <div className="textArea">
+                <strong>Please type presset`&#39;`s name</strong>             
+                <input type="text"></input> 
+                <ins>Errors/success</ins>
+            </div>
+            <div className="pressetesButtons">
+                <button className="pressetesButton" id="savePressetesButton">Save</button>
+                <button className="pressetesButton" id="cancelPressetesButton" onClick={props.onclick}>Cancel</button>
```

Comment:
In your "modalSaveWindow" component you have both - close icon and "cancel" button, I think it would be better to choose one element if they have the same option.

Reply:
rewrited